src/components/Tasks/TaskEdit.tsx

```diff
@@ -92,6 +89,10 @@ class TaskEdit extends React.PureComponent<PropsType> {
       this.state.location = event.location ? event.location : '';
       this.state.description = event.description ? event.description : '';
       this.state.timezone = event.timezone;
+      this.state.priority = event.priority ? event.priority : TaskPriorityType.None;
```

Comment:
I know the rest of the code uses the old syntax (because it's a fairly new typescript feature), but you can now just do: `this.state.priority = event.priority ?? TsakPrioritiyType.None`